src/components/ExistingProofDialog.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <v-dialog>
+    <v-card>
+      <v-card-title>
+        {{ $t('ExistingProofDialog.ChooseLatestProofs') }} <v-btn style="float:right;" variant="text" density="compact" icon="mdi-close" @click="close"></v-btn>
+      </v-card-title>
+      <v-divider></v-divider>
+      <v-card-text>
+        <v-row>
+          <v-col cols="6" sm="6" md="3" v-for="proof in userProofList" :key="proof">
+            <ProofCard :proof="proof" :reducedProofFooter="true" :isSelectable="true" :isSelected="selectedProof === proof" @proofSelected="selectProof"></ProofCard>
+          </v-col>
+        </v-row>
+    </v-card-text>
+    </v-card>
+    <v-bottom-sheet v-model="showConfirmationPopup">
```

Comment:
currently the confirmation popups (delete a price, delete a proof) are `v-dialog`. I would have kept this instead of creating a new behavior with the `v-bottom-sheet` + complexifying the `ProofCard` when selected :thinking:

Reply:
That was to be able to see the whole proof before validating. With a `v-dialog`, one can't see the proof before validating

Reply:
v-dialog has a fullscreen option : https://vuetifyjs.com/en/api/v-dialog/#props-fullscreen

Reply:
But honestly for a v1 I wouldn't even have a confirmation dialog (it's first and foremost for deletion, here it's... friction ?)
- worst case is that the user goes back to the "select a proof" 
- the user can zoom with 2 fingers
- 1 row of proofs on small screens